org.remmina.Remmina.json

```diff
@@ -0,0 +1,494 @@
+                        },
+                        {
+                            "type": "shell",
+                            "commands": [
+                                "# FTBFS fix",
+                                "sed -e '/LIBINDICATOR_LIBS/ s/\\$LIBM/ $LIBM/' -i configure.ac"
+                            ]
+                        }
+                    ]
+                }
+            ]
+        },
+        {
+            "name": "avahi",
+            "cleanup": [
+                "/bin",
+                "/lib/avahi",
+                "/share/applications/*.desktop",
+                "/share/avahi"
+            ],
+            "config-opts": [
+                "--with-distro=none",
+                "--disable-gobject",
+                "--disable-introspection",
+                "--disable-qt3",
+                "--disable-qt4",
+                "--disable-gtk",
+                "--disable-libdaemon",
+                "--disable-python",
+                "--disable-pygobject",
+                "--disable-mono",
+                "--disable-monodoc",
+                "--disable-autoipd",
+                "--disable-doxygen-doc",
+                "--disable-doxygen-dot",
+                "--disable-doxygen-xml",
+                "--disable-doxygen-html",
+                "--disable-manpages",
+                "--disable-xmltoman"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://www.avahi.org/download/avahi-0.7.tar.gz",
+                    "sha256": "57a99b5dfe7fdae794e3d1ee7a62973a368e91e414bd0dfa5d84434de5b14804"
+                }
+            ]
+        },
+        {
+            "name": "libxkbfile",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://xorg.freedesktop.org/releases/individual/lib/libxkbfile-1.0.9.tar.bz2",
+                    "sha256": "51817e0530961975d9513b773960b4edd275f7d5c72293d5a151ed4f42aeb16a"
+                }
+            ]
+        },
+        {
+            "name": "nxproxy",
+            "buildsystem": "simple",
+            "build-commands": [
+                "make CONFIGURE='./configure --prefix=/app'",
+                "# Install target does a `gzip /app/share/man/man1/*.1`",
+                "rm -rf /app/share/man",
+                "make PREFIX=/app install",
+                "sed -e 's|/usr/lib/nx|/app/lib/nx|g' -i /app/bin/nxproxy"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://code.x2go.org/releases/source/nx-libs/nx-libs-3.5.0.33-lite.tar.gz",
+                    "sha256": "e43824f820f8d307207aa4686c68f9b0808909a6b6cd2c7d75b751b14dd3129f"
+                }
+            ]
+        },
+        {
+            "name": "freerdp",
+            "buildsystem": "cmake",
```

Comment:
`cmake-ninja`

Reply:
Thank you for your review, by the way, what is the difference between `cmake` and `cmake-ninja`?

Reply:
`cmake` generates Makefiles, `ninja` is simply faster than `make` so when it works use it.

Reply:
Alright :man_facepalming: